### PR TITLE
Add Readme content

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+3. Clone your fork to your local machine in the ```/modules``` directory of your PrestaShop installation.
+4. Create a branch in your local clone of the module for your changes.
+5. Change the files in your branch. Be sure to follow [the coding standards][1]!
+6. Push your changed branch to your fork in your GitHub account.
+7. Create a pull request for your changes **on the _'dev'_ branch** of the module's project. Be sure to follow [the commit message norm][2] in your pull request. If you need help to make a pull request, read the [Github help page about creating pull requests][3].
+8. Wait for one of the core developers either to include your change in the codebase, or to comment on possible improvements you should make to your code.
+
+That's it: you have contributed to this open-source project! Congratulations!
+
+[1]: http://doc.prestashop.com/display/PS16/Coding+Standards
```
Comment (PierreRambaud) [inline Readme.md]: PS16?

Comment (matks) [reply Readme.md]: That's the links used in most of other native modules

Comment (MathiasReker) [reply Readme.md]: Readme from gsitemap: https://raw.githubusercontent.com/PrestaShop/gsitemap/master/Readme.md

**Comment by eternoendless** [comment]
Closing because obsolete

**Pull request closed without merging**